Sources/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -41,6 +41,16 @@ class AttachmentDTO: NSManagedObject {
     
     @NSManaged var message: MessageDTO
     @NSManaged var channel: ChannelDTO
+
+    override func willSave() {
+        super.willSave()
+
+        // When attachment changed, we need to propagate this change up to holding message
```

Comment:
Can you add a test for this, please?

Reply:
This behavior is already tested by `MessageSender`, e.g. in `test_sender_sendsMessage_withBothAttachmentSeedsAndUploadedAttachments`

Reply:
I see. Seems to me that having an extra test here for this specific behavior will be beneficial for future use cases. If you don't mind adding it it would great.